include/curl/multi.h

```diff
@@ -436,7 +442,7 @@ CURL_EXTERN CURLMcode curl_multi_assign(CURLM *multi_handle,
  *
  * Returns: NULL on failure, otherwise a CURL **array pointer
  */
-CURL_EXTERN CURL **curl_multi_get_handles(CURLM *multi_handle);
+CURL_EXTERN CURL **curl_multi_get_handles(CURLM *multi_handle, CURLHND filter);
```

Comment:
I prefer we call it 'flags' and make it more open ended. Probably as an 'int' instead of a typedef.

Reply:
Or even unsigned int, to give us 32 bits of flags.